src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.jsx

```diff
@@ -169,11 +169,10 @@ class GroupActivity extends React.Component {
                       date={item.dateCreated}
                       header={
                         <GroupActivityItem
-                          organization={organization}
```

Comment:
organization and projectId (slug) were not being used

Reply:
:+1: Didn't find any spreads in this component either.

Reply:
yeah, me neither 🙂